Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM php:5.6
+
+RUN echo "date.timezone = UTC" > /usr/local/etc/php/conf.d/date.ini
+
+VOLUME /app
```

Comment:
I would prefer use `/srv` instead of `/app` because the `/srv` was created for :
> Site-specific data which are served by the system.
[source](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard)